src/switch/_switch.scss

```diff
@@ -0,0 +1,68 @@
+.switch {
+  --switch-controller-bg: lightgreen;
+  --switch-active-slider-bg: white;
+  --switch-inactive-slider-bg: darkgrey;
+
+  position: relative;
+
+  display: inline-block;
+
+  width: 60px;
+  height: 34px;
+
+  &--is-disabled {
+    cursor: not-allowed;
+    pointer-events: none;
+  }
```

Comment:
`    cursor: not-allowed;`
This is not working for some reason, can you check it please? See the screenshots below:
When i hover disabled switch, cursor is not changed

Disabled switch:
![Screen Shot 2021-04-23 at 12 37 16 PM](https://user-images.githubusercontent.com/22727931/115852340-a34c3c80-a430-11eb-9ff7-d4a3edf8b466.png)

Disabled button:
![Screen Shot 2021-04-23 at 12 37 27 PM](https://user-images.githubusercontent.com/22727931/115852360-a9dab400-a430-11eb-9771-961888e411b0.png)

Reply:
When I passed `pointer-event: none;` property to `.switch__slider` for the disabled state, it works! 🚀